diff --git a/ar.xjuan.Cambalache.json b/ar.xjuan.Cambalache.json
index 0b2d08f015c23ef54be6de692b68e9076777d88b..4b8252256d9ae88b47b829de89d0a2698827ccc7 100644
--- a/ar.xjuan.Cambalache.json
+++ b/ar.xjuan.Cambalache.json
@@ -1,7 +1,7 @@
 {
     "app-id" : "ar.xjuan.Cambalache",
     "runtime" : "org.gnome.Platform",
-    "runtime-version" : "42",
+    "runtime-version" : "master",
     "sdk" : "org.gnome.Sdk",
     "separate-locales": false,
     "command" : "cambalache",
diff --git a/cambalache/cmb_view.py b/cambalache/cmb_view.py
index 7eb8852e3404c3eb7a22e6fff42ddba31c62a67d..d69010a532b02ec65ef207bdfec72a085c6cacbc 100644
--- a/cambalache/cmb_view.py
+++ b/cambalache/cmb_view.py
@@ -28,7 +28,7 @@ import socket
 import time
 
 gi.require_version('Gtk', '3.0')
-gi.require_version('WebKit2', '4.0')
+gi.require_version('WebKit2', '4.1')
 from gi.repository import GObject, GLib, Gio, Gdk, Gtk, WebKit2
 
 from . import config
--- cambalache/meson.build.orig	2023-03-22 14:47:29.613978834 -0400
+++ cambalache/meson.build	2023-03-22 14:48:17.528788327 -0400
@@ -21,7 +21,7 @@
     ['gi.repository.GLib', '>= 2.66.0'],
     ['gi.repository.Gtk', '>= 3.24.0', '3.0'],
     ['gi.repository.Gtk', '>= 4.0.0', '4.0'],
-    ['gi.repository.WebKit2', '>= 2.32.0', '4.0'],
+    ['gi.repository.WebKit2', '>= 2.32.0', '4.1'],
     ['gi.repository.Handy'],
     ['gi.repository.Adw'],
 ]
